charts/nexus-repository-manager/values.yaml

```diff
@@ -173,6 +169,3 @@ serviceAccount:
   # The name of the service account to use.
   # If not set and create is true, a name is generated using the fullname template
   name: ""
-
-psp:
-  create: false
```

Comment:
PodSecurityPolicy stuff is all gone. 🗑️

Reply:
I was just wondering why all this PSP stuff is gone? Started to playing around with the Helm chart and on our K8s cluster we needed the PodSecurityPolicies deployed to make it work. I haven't looked too much into it, just psp.create: true and the pod deployed, before that it did not.

Reply:
On which version of kubernetes are you installing? We've been trying to eliminate beta APIs and found this one to have been deprecated in k8s 1.21 with plans to remove it k8s 1.25, so we tried outright removing it. It seemed to be unneeded now, but do you have other PodSecurityPolicies with which our old config was interacting?

Reply:
Ah, it looks like maybe that `psp.create` config was controlling more than it should have been. It looks like I could have kept the Role and ClusterRole resources around and only eliminated the PodSecurityPolicy. Do you think that would be more appropriate for your use-case?

Reply:
Looking at it a bit more, I think I can bring back the PSP stuff for a while longer until it's completely removed from k8s and there's a more clear future with the Pod Security Admission.

Reply:
We run on k8s 1.20 at the moment. I hardly can remember the error message anymore, I will have a look into it tomorrow. What I can tell is we have a PSP deployed on our clusters to make sure no Pods are running as root. My guess would be we need to add nexus.securityContext.runAsUser: 200 to the values.yaml and we would be fine without all the psp stuff (the nexus user in the container has uid 200, right?).

Suggestion to either enable/disable PodSecurityContext configuration on the deployment:
values.yaml:
```
podSecurityContext:
  enable: true
  runAsUser: 200
  fsGroup: 200
```
deplyoment.yaml
```
spec:
  template:
    spec:
      {{- if .Values.nexus.securityContext.enabled }}
      securityContext:
      {{- toYaml .Values.nexus.securityContext | nindent 8 }}
      {{- end }}
```

But as I put more thought into this, having an application deploying a PSP which is a cluster-level resource is either way nogo - or do I miss something here? So I would say it is good all this PSP stuff is gone since it should in no way be deployed for a single application - 'cause it actually counts for all pods on the cluster. I will test the solution I suggested tomorrow and get back to you with my findings.